docker/Dockerfile.dev

```diff
@@ -10,7 +10,7 @@
 #       For reference:
 #           https://docs.docker.com/develop/develop-images/build_enhancements/
 
-ARG BASE_IMAGE=ubuntu:rolling
+ARG BASE_IMAGE=ubuntu:24.04
```

Comment:
During the upstream PR you will have to explain specifically why these changes to the Dockerfile are required

Reply:
@samutamm can you provide context to this change?

Reply:
Sure! Right, `rolling` points to latest Ubuntu version and even there was a problem with a specific version, one can the `BASE_IMAGE` is already an argument that can be switched to more suitable version. Let's revert this change.

Reply:
I mean at least we'll need to use a ROCm supported distro https://rocm.docs.amd.com/en/latest/compatibility/compatibility-matrix.html

Not sure on the context of serve choosing rolling.

Reply:
Very good point, that alone rules out using the rolling release which is now 24.10.